src/config.rs

```diff
@@ -46,3 +46,16 @@ pub fn zo_maxage() -> Result<Rank> {
         None => Ok(1000.0),
     }
 }
+
+pub fn zo_follow_symlinks() -> bool {
+    match env::var_os("_ZO_FOLLOW_SYMLINKS") {
+        Some(follow_osstr) => {
+            let follow_str = follow_osstr.to_str();
+            match follow_str.unwrap_or("1").to_lowercase().as_ref() {
+                "0" | "false" | "no" => false,
+                _ => true,
+            }
+        }
+        None => true,
+    }
+}
```

Comment:
I think it would be much clearer to have environment variables default to the false value. Additionally, checking for multiple values brings additional complexity, and would not be consistent with the other environment variables.

I think we should change the name to something like `_ZO_RESOLVE_SYMLINKS`, which defaults to false unless explicitly set to `1`.